gradle-palantir-java-format/src/test/groovy/com/palantir/javaformat/gradle/PalantirJavaFormatSpotlessPluginTest.groovy

```diff
@@ -57,6 +57,10 @@ class PalantirJavaFormatSpotlessPluginTest extends IntegrationTestKitSpec {
                      classpath 'com.palantir.baseline:gradle-baseline-java:6.21.0'
                      classpath 'com.palantir.gradle.jdks:gradle-jdks:0.62.0'
                      classpath 'com.palantir.gradle.jdkslatest:gradle-jdks-latest:0.17.0'
+
+                     constraints {
```

Comment:
just to double check, if you reverted your changes, did you see the test fail?

Reply:
I've added the SupportsSpotless622 test, which fails for develop "Not allowed to resolve configuration ':palantirJavaFormat' at configuration time", but passes with this PR.